streams/src/test/java/org/apache/kafka/streams/perf/YahooBenchmark.java

```diff
@@ -68,23 +62,24 @@
     static class ProjectedEvent {
         /* attributes need to be public for serializer to work */
         /* main attributes */
-        public String eventType;
-        public String adID;
+        String eventType;
+        String adID;
 
         /* other attributes */
-        public long eventTime;
-        public String userID = UUID.randomUUID().toString(); // not used
-        public String pageID = UUID.randomUUID().toString(); // not used
-        public String addType = "banner78";  // not used
-        public String ipAddress = "1.2.3.4"; // not used
+        long eventTime;
+        //public String userID = UUID.randomUUID().toString(); // not used
```

Comment:
Is this intentional? If they are retained as comments just for context information I'd suggest use `/**/` as a nit comment.